test/moonwall.config.json

```diff
@@ -557,7 +557,10 @@
         {
             "name": "zombie_dancebox_upgrade",
             "testFileDir": ["suites/zombie_dancebox_upgrade"],
-            "runScripts": ["downloadPolkadot.ts"],
+            "runScripts": [
+                "download-latest-rt-binaries.sh",
```

Comment:
You still need `downloadPolkadot.ts` here, and in all zombienet jobs if they don't have it

Reply:
Yes, seems like this. I had it locally. Will add it as well.